repose-aggregator/components/filters/client-auth/src/main/java/org/openrepose/filters/clientauth/atomfeed/AuthFeedReader.java

```diff
@@ -1,7 +1,8 @@
 package org.openrepose.filters.clientauth.atomfeed;
 
+import org.openrepose.services.serviceclient.akka.AkkaServiceClientException;
+
 public interface AuthFeedReader {
-   
-   CacheKeys getCacheKeys();
-   String getFeedId();
```

Comment:
It was annotated by IntelliJ as not being used, so while I was there I cleaned it up.